pyproject.toml

```diff
@@ -9,10 +9,10 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.8.1"
 langchain = ">=0.0.333"
-langsmith = "^0.0.64"
-tqdm = "^4.66.1"
-ipywidgets = "^8.1.1"
-tabulate = "^0.9.0"
+langsmith = ">=0.0.64"
```

Comment:
Wanna push this to 0.0.66? adds in the test run dataframe to the client

Reply:
i don't want to push the requirement, but i updated it in the lock file

Reply:
unless okay to force users to upgrade?